recipes/KWN-deform/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -ex
+
+# Variables not set for MSYS2 toolchain
+FC="${FC:-gfortran}"
+AR="${AR:-ar}"
+
+
+export FPM_FC="${FC}"
+export FPM_AR="${AR}"
+export FPM_LDFLAGS="${LDFLAGS}"
+export FPM_FFLAGS="-fbounds-check -ffree-line-length-0 -fimplicit-none -O3 -DWITH_QP=1"
+fpm build --tests
+fpm test
+fpm install --prefix "${PREFIX}"
```

Comment:
These looks good to me but I'm not an expert on modern fpm. @isuruf I'd love if you could take a quick look here before we merge it.

Reply:
@ocefpaf & @isuruf - have you had chance to check these for us?

Reply:
If @isuruf can't check this today I'll merge it this afternoon and we'll see how this perform.

Reply:
Would it be possible to merge this in to conda forge now?